Users/binah/access-control/access-control.cs

```diff
@@ -0,0 +1,320 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Sansar.Script;
+using Sansar.Simulation;
+using System.Diagnostics;
+
+// register agents on entry
+
+public class AccessControl : SceneObjectScript
```

Comment:
You might benefit from extending `LibraryBase`, you get debug api out of the the box that is togglable from a central script.

Reply:
Thanks for the tip I didn't know that.